src/Microsoft.CodeAnalysis.Razor.Workspaces/AcceptedCharacters.cs

```diff
@@ -3,7 +3,7 @@
 
 using System;
 
-namespace Microsoft.VisualStudio.LanguageServices.Razor
+namespace Microsoft.CodeAnalysis.Razor
```

Comment:
Due to ^^^^ this issue, I'm not sure that your strategy here is going to work VS just copy-pasting services in the `Microsoft.CodeAnalysis.Razor` namespace.

I want to make sure that when we make these changes that we are **done** and don't have to keep asking Barry's team to integrate more small changes.